source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
```

Comment:
BTW. I am just inviting three new committers to Airflow so I will test out those new templates.